internal/cli/atlas_whitelist_delete.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
```

Comment:
[q] This is coming up as 404 not found. Do we need it? I wonder how's it's working still.

Reply:
This is go mod versioning system, this is the repo, https://github.com/AlecAivazis/survey, and the v2 just makes go mods to look for a v2.x.x branch, in theory all repos built for go mods should now version like this, some are already doing `v5` and such

Reply:
Got it, thanks for the explanation!